### PR TITLE
Amin/frontend/skeleton/admin tools

### PR DESCRIPTION
**Pull request by AminesCodes**
Admin CRUD for skills, cohorts and users
## Description

Allow admin users to get/add/edit/delete skills, cohorts and users
## Motivation and Context

Part of the admin management tools
## How Has This Been Tested?

Frontend + Postico
## Types of changes

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

File: backend/queries/volunteers.js [hunk 1]
```diff
@@ -94,7 +94,7 @@ const addVolunteer = async (user, password) => {
     const volunteer = await db.one(insertQuery, user);
 
     const promises = [];
-    user.skills.map(skillId => promises.push(db.none('INSERT INTO volunteer_skills (volunteer_id, skill_id) VALUES ($1, $2)', [user.userId, skillId])));
+    user.skills.map(skillId => promises.push(db.none('INSERT INTO volunteer_skills (volunteer_id, skill_id) VALUES ($1, $2)', [volunteer.v_id, skillId])));
```
Comment (joseph-p-pasaoa) [inline backend/queries/volunteers.js:97]: same notes as older p.req

File: backend/routes/cohorts.js [hunk 2]
```diff
@@ -30,47 +30,59 @@ const getAllCohorts = async (req, res, next) => {
...
-    const cohort = processInput(req.body.cohort, "hardVC", "cohort name", 100);
-
-    const response = await queries.insertCohort(cohort);
-    res.status(201);
-    res.json({
-        status: "success",
-        message: `new cohort '${cohort}' added`,
-        payload: response
-    });
+    if (req.user || req.user.a_id) {
```
Comment (joseph-p-pasaoa) [inline backend/routes/cohorts.js:33]: are these supposed to be && not ||?

File: frontend/src/Components/Feedback.jsx [hunk 3]
```diff
@@ -3,19 +3,19 @@ import React from 'react'
 export default (props) => {
```
Comment (joseph-p-pasaoa) [inline frontend/src/Components/Feedback.jsx:3]: same comment as previous p.req

File: frontend/src/Components/Profile/VolunteerProfile.jsx [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+        props.setTitle(loggedUser.title);
+        props.setMentor(loggedUser.mentoring);
+        props.setOfficeHours(loggedUser.office_hours);
+        props.setTechMockInterview(loggedUser.tech_mock_interview);
+        props.setBehavioralMockInterview(loggedUser.behavioral_mock_interview);
+        props.setProfessionalSkillsCoach(loggedUser.professional_skills_coach);
+        props.setHostSiteVisit(loggedUser.hosting_site_visit);
+        props.setIndustrySpeaker(loggedUser.industry_speaker);
+        getVolunteerSkills();
+    }, []);
```
Comment (joseph-p-pasaoa) [inline frontend/src/Components/Profile/VolunteerProfile.jsx:65]: i think the loggedUser needs to go in the dep array here

File: backend/queries/volunteers.js (hunk 1 above)
Comment (AminesCodes) [reply backend/queries/volunteers.js:97]: I will switch to forEach, good catch
but as explained in the answer for the previous PR, I don't see the use of sending both data
we can discuss this during our meeting later one this evening

File: backend/routes/cohorts.js (hunk 2 above)
Comment (AminesCodes) [reply backend/routes/cohorts.js:33]: yes!! thank you

**Pull request merged**